app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,27 @@
 module Api
   class AuthenticationsController < BaseController
+    def create_resource(_type, _id, data)
+      attrs = validate_auth_attrs(data)
+      klass = ::Authentication.class_from_request_data(attrs)
+      # TODO: Temporary validation - remove
+      raise 'type not currently supported' unless klass.respond_to?(:create_in_provider_queue)
+      task_id = klass.create_in_provider_queue(attrs['manager_resource'], attrs.except('type', 'manager_resource'))
+      action_result(true, 'Creating Authentication', :task_id => task_id)
+    rescue => err
+      action_result(false, err.to_s)
+    end
+
+    private
+
+    def validate_auth_attrs(data)
+      raise 'must supply a manager resource' unless data['manager_resource']
+      attrs = data.dup
+      attrs['manager_resource'] = if data['manager_resource'].key?('href')
+                                    parse_href(data['manager_resource']['href']).last
+                                  elsif data['manager_resource'].key?('id')
+                                    data['manager_resource']['id']
+                                  end
```

Comment:
@abellotti good question.

It's an ID, and for what we're starting with [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script.rb#L7), it is an `ExtManagementSystem`.

I originally limited it to `providers`.

I believe @jameswnl mentioned that technically, a manager can be a different type of resource. Is that correct? And @jameswnl - do you think we can send an object to `create_in_provider_queue` rather than the ID?